### PR TITLE
feature(ui-ux): add banner

### PR DESCRIPTION
**Pull request by JoshuaC817**
#### What this PR does / why we need it:
Added a top banner to header

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] No console errors on web
- [ ] Tested on Light mode and Dark mode\*
- [x] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*
- [ ] Added translations\*

### CODE REVIEW

File: apps/web/src/components/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      <div className="pl-4">
+        <span className="inline-block md:inline">
+          Make sure you are visiting&nbsp;
+        </span>
+        <span className="inline font-semibold text-transparent bg-clip-text bg-gradient-to-r from-[#42F9C2] to-[#082FD4]">
+          https://quantumbridge.app&nbsp;
+        </span>
+        – check the URL correctly.&nbsp;
+        <span className="md:inline-block lg:inline">
+          This is still in Beta. Use at your own risk.
```
Comment (thedoublejay) [inline apps/web/src/components/Banner.tsx:18]: Let's remove this first

File: apps/web/src/components/Banner.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <div className="flex flex-row items-center py-[18px] text-dark-1000 bg-gradient-to-r from-[#00000066] to-[#00000000] px-6 lg:px-[120px] md:px-10 text-xs md:text-sm">
+      <div className="h-4 w-4 min-w-4">
+        <HiLockClosed size={16} />
+      </div>
+      <div className="pl-4">
+        <span className="inline-block md:inline">
+          Make sure you are visiting&nbsp;
+        </span>
+        <span className="inline font-semibold text-transparent bg-clip-text bg-gradient-to-r from-[#42F9C2] to-[#082FD4]">
+          https://quantumbridge.app&nbsp;
```
Comment (thedoublejay) [inline apps/web/src/components/Banner.tsx:14]: Also can we make this a href

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: apps/web/src/components/Banner.tsx (hunk 2 above)
Comment (JoshuaC817) [reply apps/web/src/components/Banner.tsx:17]: Done

File: apps/web/src/components/Banner.tsx (hunk 1 above)
Comment (JoshuaC817) [reply apps/web/src/components/Banner.tsx]: Done

**Pull request merged**